src/App.js

```diff
@@ -80,11 +77,12 @@ class App extends Component {
         />
 
         <NewUser
-          showModal={() => this.showUserModal()}
+          showModal={!this.state.username}
```

Comment:
The syntax that leads a value with a `!`bang, is a check to see if there is a value, if there is not a value then it returns false, if there is a value it returns true. Remember the `showModal` needs to be a boolean to toggle on and off. This is my attempt to create a boolean in less lines of code.